packages/ethers/e2e-tests/simple-account.test.ts

```diff
@@ -37,13 +37,17 @@ describe("Simple Account Tests", async () => {
   });
 
   it("should execute successfully", async () => {
+    // TODO: this is super hacky right now for Ethers
+    // we have to wait for the test above to run and be confirmed so that this one submits successfully using the correct nonce
+    // one way we could do this is by batching the two UOs together
+    await new Promise((resolve) => setTimeout(resolve, 7500));
```

Comment:
we can't do the `waitForUserOperationTransaction` thing here too?

Reply:
sadly no since the ethers `AccountSigner` doesn't implement a `waitForUserOperationTransaction`. Should we add that implementation?